modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets_{context}"]
+= Creating a secret using the web console
+
+As an administrator, you can create secrets using the web console. 
```

Comment:
A user does not necessarily have to be a namespace/cluster administrator to create secrets.